webpack.sample.config.js

```diff
@@ -192,12 +192,15 @@ var webpackConfig = {
 			test: /\.css$/,
 			loader: 'style-loader!css-loader'
 		}, {
-			test: /\.(png|jpg|jpeg|gif|svg)$/,
+			  test: /\.(png|jpg|jpeg|gif|svg|eot|ttf|woff|woff2)$/,
 			loader: 'url'
 		}, {
 			test: /\.html$/,
 			loader: 'html'
-		}],
+		}, {
+			test: /\.scss$/,
+			loaders: ["style-loader", "css-loader", "sass-loader"]
+	}],
 	},
```

Comment:
why make changes to the sample config and not the actual webpack.config file?

Reply:
The `webpack.config` file is included in the `gitignore`, so `webpack.sample.config` is provided as a generic file. This file is supposed to be copied into the `webpack.config` and then customize the paths according to your file system.

Reply:
I understand this, but since the changes are limited to just our project for now, shouldn't this then go into the ```webpack.config```

Reply:
No, you will need to copy them into your `webpack.config`.